.gitignore

```diff
@@ -21,3 +21,4 @@ __pycache__/
 htmlcov/
 .tox/
 .ruff_cache/
+.python-version
```

Comment:
In general, we are creating a lot of code here that will need to be supported. It's not clear yet which parts of the code will be necessary in the Python runtime. For example, the conversion handler can be used from JS/TS by providing both simple and categorized methods in JSON instead of duplicating the not-so-final conversion class instance in Python.